resources/test/dumps/Bacteria Hunt Speeds.js

```diff
@@ -165,7 +168,15 @@ var procedures = (function() {
           procedures["MAKE-FLAGELLA"]();
           SelfManager.self().setXY(Prims.randomCoord(world.topology.minPxcor, world.topology.maxPxcor), Prims.randomCoord(world.topology.minPycor, world.topology.maxPycor));
         }, true);
-      }), [1, 2, 3, 4, 5, 6]);
+      }, "[ this-variation ->
```

Comment:
I think I follow the thought process here, but I'd be interested to know what problems you're predicting from this.  My intuition is that this can't really cause problems in NLW except in widgets, and then maybe the escaping should only be handled there.  Either way, seems to me that this concern is not exclusive to anonprocs—strings are probably the most concerning in this regard.

I'm not opposed to HTML-escaping the strings, but I do have concerns about us doing the escaping too soon, since doing so can actually change the behavior of a model (e.g. `let f [x -> x] show ifelse-value (length (word "" f) <= 8)
 [ "shorty" ] [ "lengthy" ]` could lead to different results across NLW and NL if we HTML-escape within the compiler or engine).